src/ClientGenerator/ClientGenerator.csproj

```diff
@@ -14,7 +14,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Orleans.CodeGeneration</RootNamespace>
     <AssemblyName>ClientGenerator</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <PublishUrl>publish\</PublishUrl>
     <Install>true</Install>
```

Comment:
Indeed. This is how the cruft builds up over time in these proj files and VS in general.  :(

You can see exactly why lots of projects now use FAKE or RAKE or SBT for simpler build scripts these days. Maybe DNX will help reboot things?

I bet you never peaked under the covers of the "HTML" that MS Word "generated" back in the day and decided to use something else instead? :blush: